build_library/grub_install.sh

```diff
@@ -183,6 +183,12 @@ sudo grub-mkimage \
     --output "${ESP_DIR}/${GRUB_DIR}/${CORE_NAME}" \
     "${CORE_MODULES[@]}"
 
+if [[ "${FLAGS_target}" != i386-pc ]]; then
+    for mod in "${CORE_MODULES[@]}"; do
+      rm "${ESP_DIR}/${GRUB_DIR}/${mod}.mod"
```

Comment:
`Command 'rm "${ESP_DIR}/${GRUB_DIR}/${mod}.mod"' exited with nonzero code: 1`
`rm: cannot remove '/tmp/tmp.WM8QCKnE6l/flatcar/grub/arm64-efi/normal.mod': Permission denied`
 -> looks that this commands needs also a `sudo`?

Reply:
Added a sudo, and the build now works.